backstory/src/test/java/com/google/sps/story/StoryManagerTest.java

```diff
@@ -85,6 +85,25 @@ public void invalidTemperature() {
     storyManager = new StoryManagerImpl(PREFIX_SAMPLE, SIZE_SAMPLE, 1000.0);
   }
 
+  @Test
+  public void cyclePastBounds() {
+    storyManager = new StoryManagerImpl(PREFIX_SAMPLE, SIZE_SAMPLE, TEMPERATURE_SAMPLE);
+    try{
```

Comment:
I think you're supposed to leave a space here.

```suggestion
    try {
```